src/browser/BrowserAction.cpp

```diff
@@ -516,6 +516,74 @@ QJsonObject BrowserAction::handleGlobalAutoType(const QJsonObject& json, const Q
     return buildResponse(action, browserRequest.incrementedNonce);
 }
 
+#ifdef WITH_XC_BROWSER_PASSKEYS
+QJsonObject BrowserAction::handlePasskeysGet(const QJsonObject& json, const QString& action)
+{
+    if (!m_associated) {
+        return getErrorReply(action, ERROR_KEEPASS_ASSOCIATION_FAILED);
+    }
+
+    const auto browserRequest = decodeRequest(json);
+    if (browserRequest.isEmpty()) {
+        return getErrorReply(action, ERROR_KEEPASS_CANNOT_DECRYPT_MESSAGE);
+    }
+
+    const auto command = browserRequest.getString("action");
+    if (command.isEmpty() || command.compare(BROWSER_REQUEST_PASSKEYS_GET) != 0) {
+        return getErrorReply(action, ERROR_KEEPASS_INCORRECT_ACTION);
+    }
+
+    const auto publicKey = browserRequest.getObject("publicKey");
+    if (publicKey.isEmpty()) {
+        return getErrorReply(action, ERROR_PASSKEYS_EMPTY_PUBLIC_KEY);
+    }
+
+    const auto origin = browserRequest.getString("origin");
+    if (!origin.startsWith("https://")) {
+        return getErrorReply(action, ERROR_KEEPASS_ACTION_CANCELLED_OR_DENIED);
```

Comment:
I think this is the wrong error. Either ERROR_KEEPASS_NO_URL_PROVIDED or create a new one specifically for PassKeys like: ERROR_PASSKEYS_INVALID_URL_PROVIDED